website/docs/r/secure_okta_ml_policy.md

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "Sysdig Secure"
+layout: "sysdig"
+page_title: "Sysdig: sysdig_secure_okta_ml_policy"
+description: |-
+  Retrieves a Sysdig Secure Okta ML Policy.
+---
+
+# Resource: sysdig_secure_okta_ml_policy
+
+Retrieves the information of an existing Sysdig Secure Okta ML Policy.
+
+-> **Note:** Sysdig Terraform Provider is under rapid development at this point. If you experience any issue or discrepancy while using it, please make sure you have the latest version. If the issue persists, or you have a Feature Request to support an additional set of resources, please open a [new issue](https://github.com/sysdiglabs/terraform-provider-sysdig/issues/new) in the GitHub repository.
+
+## Example Usage
+
+```terraform
+resource "sysdig_secure_okta_ml_policy" "policy" {
+  name        = "Test Okta ML Policy 1"
+  description = "Test Okta ML Policy Description"
+  enabled     = true
+  severity    = 4
+
+  rule {
+    description = "Test Okta ML Rule Description"
+
+    anomalous_console_login {
+      enabled   = true
+      threshold = 1
+    }
+}
+```
```

Comment:
You are missing a `}` in the example: 

```suggestion
```terraform
resource "sysdig_secure_okta_ml_policy" "policy" {
  name        = "Test Okta ML Policy 1"
  description = "Test Okta ML Policy Description"
  enabled     = true
  severity    = 4

  rule {
    description = "Test Okta ML Rule Description"

    anomalous_console_login {
      enabled   = true
      threshold = 1
    }
  }
}
```
```